app/javascript/css/application.css

```diff
@@ -20,4 +20,6 @@
 
 body {
     @apply font-body bg-bgGray text-textBase antialiased;
+    line-height: 1;
+    font-size: 14px;
```

Comment:
My first instinct would've been to use Tailwind's built-in `.text-base` class for `font-size: 1rem`, and then set rem base in `html` and `body`: https://snook.ca/archives/html_and_css/font-size-with-rem

Using textBase to describe font color might get confusing here. I'm typically using the language `tiny`, `small`, `base`, `large`, `extra-large` for sizing things.

Reply:
@taiyab We're using `px` rather than `rem` too, as the value of `rem` has disappeared in modern browsers, and px are easier to reason about. So I've removed all of those rem classes from tailwind anyway, so we can set styles to be whatever (e.g. base case be 14px, large can be 16px, etc).

Reply:
Sure, I'm just wary of eliminating what makes it good in the first place (default set of utility classes you can quickly apply to everything, and light component styling).

Regardless, I'd adjust to any way of writing it.

Reply:
Yeah, sure - I am using it everywhere else to be clear. And I totally could use it here too! My point more was that I want to choose what our utility classes are, rather than use some arbitary default ones. What you've chosen as the classes is what I'll use here, exactly in the same way as what we've done with box-shadows below.

Reply:
```suggestion
    @apply fontSize-14;
```